src/sass/partials/_advantages.scss

```diff
@@ -0,0 +1,32 @@
+.advantages {
+  background-color: $color-secondary-background;
+  padding-top: 39px;
+  padding-bottom: 59px;
+  padding-left: 19px;
+  display: flex;
+}
+
+.advantages-list {
```

Comment:
lista może mieć flexa, wtedy odstępy załatwisz gap: x, a nie tak\jak teraz marginami.  Chyba, że masz inny pomysł na to:)